src/passport-saml/saml.ts

```diff
@@ -8,17 +8,34 @@ import * as xmldom from 'xmldom';
 import * as url from 'url';
 import * as querystring from 'querystring';
 import * as xmlbuilder from 'xmlbuilder';
+// @ts-expect-error no typings for xml-encryption
```

Comment:
`xml-encryption` is now managed by `auth0`. They may be interested in maintaining TypeScript types in the main repo. https://github.com/auth0/node-xml-encryption

Reply:
I think it is out of the scope of this PR. I use // @ts-expect-error instead of @ts-ignore because compilation will fail when node-xml-encryption adds typings. Fixing this problem will take unknown period of time, so I suggest keeping this comment for now